src/phone-input/constants.js

```diff
@@ -5,9 +5,6 @@ This source code is licensed under the MIT license found in the
 LICENSE file in the root directory of this source tree.
 */
 // @flow
-export const DEFAULT_MAX_DROPDOWN_WIDTH = '400px';
```

Comment:
breaking change since it was exported https://github.com/uber-web/baseui/blob/master/src/phone-input/index.js#L17